DRODLib/DbXML.cpp

```diff
@@ -225,6 +244,8 @@ void CDbXML::PerformCallback(long val) {
 void CDbXML::PerformCallbackf(float fVal) {
 	if (pCallbackObject)
 		pCallbackObject->Callbackf(fVal);
+
+	const bool successIgnored = CDbXML::streamingOut.flush(10 * 1024*1024); //10 MB
```

Comment:
Extract the magic number to a constant at the top of the file perhaps?

Reply:
Good idea. Thank you. Done!